repository/src/main/java/org/alfresco/repo/domain/permissions/AclDAOImpl.java

```diff
@@ -1662,9 +1663,29 @@ public void beforeCommit(boolean readOnly)
             }
             // Update it
             long commitTimeMs = System.currentTimeMillis();
+            AlfrescoTransactionSupport.bindResource(RESOURCE_KEY_ACL_CHANGE_SET_COMMIT_TIME_MS, commitTimeMs);
             aclCrudDAO.updateAclChangeSet(changeSetId, commitTimeMs);
         }
     }
+
+    /**
+     * Retrieve the value from the current thread local, avoiding the access to the database.
+     * @return

```

Comment:
What happens if this method returns null?

Reply:
I'll change the Javadoc in the interface, if null it means that there isn't a transaction active.